src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -168,24 +173,23 @@ private async Task CreateBlocksAsync()
             }
         }
 
-        /// <summary>
-        /// Doesn't let the node progress until the given time slot.
-        /// </summary>
-        /// <param name="myTimestamp">The next time slot this node should mine at.</param>
-        protected virtual async Task WaitUntilWeCanMineAsync(uint myTimestamp)
+        private async Task<uint> WaitUntilCanMineAsync()
```

Comment:
This is clear but another suggestion is `WaitUntilMiningSlotAsync`

Reply:
Fixed